web/filament-js/filament.d.ts

```diff
@@ -624,7 +624,7 @@ export class gltfio$FilamentInstance {
     public getSkinNames(): Vector<string>;
     public attachSkin(skinIndex: number, entity: Entity): void;
     public detachSkin(skinIndex: number, entity: Entity): void;
-    public getMaterialInstances(): Vector<MaterialInstance>;
```

Comment:
Why this change here? The other getters return a Vector<>.

Reply:
*getMaterialVariantNames* is wrapper by *vectorToArray*,  which is convenient when iterate, so make this change to *getMaterialInstances* too.
What's the convention (vectorToArray or not) here, should all interfaces defined in *filament.d.ts* use Vector?

Reply:
I think we should keep this consistent with the other APIs and continue to return a `Vector<MaterialInstance>`.

Reply:
Sure, reverted.